marklogic-data-hub-central/ui/src/components/modeling/graph-view/graph-vis/graph-vis.tsx

```diff
@@ -303,10 +303,23 @@ const GraphVis: React.FC<Props> = (props) => {
 
   // Focus on the selected nodes in filter input
   useEffect(() => {
-    if (network && props.isEntitySelected) {
-      network.focus(props.entitySelected);
-    }
-  }, [network, props.isEntitySelected]);
+    let focused = true;
+    (async () => {
+      if (focused && network && props.isEntitySelected) {
+        let viewPosition: any = await network.getViewPosition();
+        await network.focus(props.entitySelected);
+        let viewPositionPayload = defaultHubCentralConfig;
```

Comment:
Let's initialize **viewPositionPayload** to **props.hubCentralConfig** here. Otherwise, it will reset the Concept config in hubCentral.json config file.